app/models/download_writer.rb

```diff
@@ -74,15 +74,18 @@ def get_calibre_command
     ### Format-specific options
     # epub: don't generate a cover image
     epub = download.file_type == "epub" ? ["--no-default-epub-cover"] : []
-    # pdf: decrease margins from 72pt default
+    # pdf:
+    # - decrease margins from 72pt default
+    # - only include necessary characters when embedding fonts
```

Comment:
Can you move the comments to just above the flags they're about? Like the "ebook-convert" array below.

Reply:
How do we want it?

Option A:
```
    ### Format-specific options
    # epub: don't generate a cover image
    epub = download.file_type == "epub" ? ["--no-default-epub-cover"] : []
    # pdf:
    pdf = []
    if download.file_type == "pdf"
      pdf = [
        # decrease margins from 72pt default
        "--pdf-page-margin-top", "36",
        "--pdf-page-margin-right", "36",
        "--pdf-page-margin-bottom", "36",
        "--pdf-page-margin-left", "36",
        "--pdf-default-font-size", "17",
        # only include necessary characters when embedding fonts
        "--subset-embedded-fonts"
      ]
    end
```

Option B:
```
    ### Format-specific options
    # epub: don't generate a cover image
    epub = download.file_type == "epub" ? ["--no-default-epub-cover"] : []
    pdf = []
    if download.file_type == "pdf"
      pdf = [
        # pdf: decrease margins from 72pt default
        "--pdf-page-margin-top", "36",
        "--pdf-page-margin-right", "36",
        "--pdf-page-margin-bottom", "36",
        "--pdf-page-margin-left", "36",
        "--pdf-default-font-size", "17",
        # pdf: only include necessary characters when embedding fonts
        "--subset-embedded-fonts"
      ]
    end
```

Or something else?

Reply:
I think either B, or A without that "# pdf:".

Reply:
I think I'll go with B -- with A without the `# pdf:` comment, it's not entirely clear we've switched from modifying just EPUBs to just PDFs